def ss():
    try:
        xx=1
        return
    except Exception as ex:
        xx=2
    finally:
        xxc=3

ss()